cloud-gateway-service/src/test/java/org/zowe/apiml/cloudgatewayservice/acceptance/xForwardHeaders/MutateRemoteAddressFilter.java

```diff
@@ -0,0 +1,42 @@
+/*
+ * This program and the accompanying materials are made available under the terms of the
+ * Eclipse Public License v2.0 which accompanies this distribution, and is available at
+ * https://www.eclipse.org/legal/epl-v20.html
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Copyright Contributors to the Zowe Project.
+ */
+
+package org.zowe.apiml.cloudgatewayservice.acceptance.xForwardHeaders;
+
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.core.Ordered;
+import org.springframework.core.annotation.Order;
+import org.springframework.web.server.ServerWebExchange;
+import org.springframework.web.server.WebFilter;
+
+import java.net.InetSocketAddress;
+
+@Configuration
+public class MutateRemoteAddressFilter {
```

Comment:
This configuration probably shouldn't be applied to all test cases. What about using a technical profile to be enabled only for the new tests?